### ISSUE TITLE
CI failure on main: documentation check

### ISSUE THREAD

**Issue by jacobtylerwalls** [pylint-dev/pylint#8762]
### Bug description

The last two commits on `main` fail [this documentation check](https://github.com/pylint-dev/pylint/actions/runs/5238083730/jobs/9456656405).

Since it began when merging #8757, I double-checked that #8757 wasn't to blame by [reverting it on a PR on my fork](https://github.com/jacobtylerwalls/pylint/pull/18); didn't help. (Makes sense, test failure didn't come up when merging the PR.)

Can't reproduce locally.

### Command used

```shell
CI main branch
```

### Pylint output

```shell
See failing action
```

### Expected behavior

CI passes (it does locally)

### Pylint version

```shell
CI main branch
```

**Comment by jacobtylerwalls** [comment]
Was able to reproduce it locally with tox instead of pytest.
